src/Components/Components/src/Routing/Router.cs

```diff
@@ -92,6 +92,7 @@ public partial class Router : IComponent, IHandleAfterRender, IDisposable
     /// over wildcards.
     /// <para>This property is obsolete and configuring it does nothing.</para>
     /// </summary>
+    [Obsolete("This property is obsolete and configuring it has not effect.")]
```

Comment:
I assume there's no recommended alternative or anything?

Reply:
No, this was something that was left there in 3.0/1, (can't remember)